fakestorage/object_test.go

```diff
@@ -604,3 +604,33 @@ func TestServerClientObjectDeleteErrors(t *testing.T) {
 		}
 	})
 }
+
+func TestServerClientObjectSetAclPrivate(t *testing.T) {
+	objs := []Object{
+		{BucketName: "some-bucket", Name: "img/public-to-private.jpg"},
+	}
+
+	runServersTest(t, objs, func(t *testing.T, server *Server) {
+		t.Run("public to private", func(t *testing.T) {
+			ctx := context.Background()
+			objHandle := server.Client().Bucket("some-bucket").Object("img/public-to-private.jpg")
```

Comment:
Here you get a handle to this object, but who created it?

Reply:
the runServersTest is an internal function that creates the objs that you pass to it